PLAYER/SwitchCrimeType.md

```diff
@@ -12,7 +12,7 @@ void _SWITCH_CRIME_TYPE(Player player, int p1);
 ```
 This was previously named as "RESERVE_ENTITY_EXPLODES_ON_HIGH_EXPLOSION_COMBO"  
 which is obviously incorrect.  
-Seems to only appear in scripts used in Singleplayer. p1 ranges from 2 - 46.  
+Seems to only appear in scripts used in Singleplayer. p1 ranges in 2 - 46.  
```

Comment:
Maybe `p1` ranges from 2 to 46 would be better suited? This would just update the imported documentation by the way. Native descriptions that are found inside code blocks are actually imported from external native sources i.e. nativedb. It would perhaps be better to figure out what `p1` actually is and document based on that? 🙂

Reply:
The native doesn't even seem to exist *anymore*. Probably new name used by R*?

Reply:
> The native doesn't even seem to exist anymore.

From `CrossMapping_Universal.h`:

```cpp
{ { 0x9A987297ED8BD838, 0x19372A1A61357555, 0x8580C7F49F4DB286, 0xD9FB629772F03712, 0xF34C83685678DA00, 0x959847D9B98A8E0E, 0xB67630FD0CAA583D, 0x2998F8A7E3956B73, 0x35E434328CEC3651, 0x810BAE16BFB69127, 0xBD322AD12FAB8AE8, 0x2907F140DB08C232, 0x31EC550CD19AFDEC, 0x62AB53D880F2DCC1, 0xBA9C14A523781E80, 0x5569805F9BA0EE5C, 0x52AA7BC2DF797B01, 0xE8EB4BCE461C6092, 0xC932A847D0064406, 0x09DAAE86322DF12D, 0x877EFFFA3185C8EF, 0xFB090DDC6C86AF36, 0x0B54B0F65E7FEED7, 0xACBA79A91C121E78, 0x65C6DBC40481CB81, 0x01E6FD194DCBEB7D, 0xC9C053A89388D54B, 0xFE89392FAC7394F2 } },
```

It doesn't seem to have been removed - what are you using as a reference to say it "doesn't exist"?

alloc8or's database refers to the command as `SUPPRESS_CRIME_THIS_FRAME`, by the way, where `p1` is `crimeType` (i.e. the enum from REPORT_CRIME):

```json
		"0x9A987297ED8BD838": {
			"name": "SUPPRESS_CRIME_THIS_FRAME",
			"jhash": "0x59B5C2A2",
			"comment": "crimeType: see REPORT_CRIME",
```

Reply:
> > The native doesn't even seem to exist anymore.
> 
> From `CrossMapping_Universal.h`:
> 
> ```c++
> { { 0x9A987297ED8BD838, 0x19372A1A61357555, 0x8580C7F49F4DB286, 0xD9FB629772F03712, 0xF34C83685678DA00, 0x959847D9B98A8E0E, 0xB67630FD0CAA583D, 0x2998F8A7E3956B73, 0x35E434328CEC3651, 0x810BAE16BFB69127, 0xBD322AD12FAB8AE8, 0x2907F140DB08C232, 0x31EC550CD19AFDEC, 0x62AB53D880F2DCC1, 0xBA9C14A523781E80, 0x5569805F9BA0EE5C, 0x52AA7BC2DF797B01, 0xE8EB4BCE461C6092, 0xC932A847D0064406, 0x09DAAE86322DF12D, 0x877EFFFA3185C8EF, 0xFB090DDC6C86AF36, 0x0B54B0F65E7FEED7, 0xACBA79A91C121E78, 0x65C6DBC40481CB81, 0x01E6FD194DCBEB7D, 0xC9C053A89388D54B, 0xFE89392FAC7394F2 } },
> ```
> 
> It doesn't seem to have been removed as there's entries in _all_ builds - what are you using as a reference to say it "doesn't exist"?
> 
> alloc8or's database refers to the command as `SUPPRESS_CRIME_THIS_FRAME`, by the way, where `p1` is `crimeType` (i.e. the enum from REPORT_CRIME):
> 
> ```json
> 		"0x9A987297ED8BD838": {
> 			"name": "SUPPRESS_CRIME_THIS_FRAME",
> 			"jhash": "0x59B5C2A2",
> 			"comment": "crimeType: see REPORT_CRIME",
> ```

Neat, updated their PR since I noticed the `jhash` matches the native name 😀